src/category_theory/glue_data.lean

```diff
@@ -0,0 +1,201 @@
+/-
+Copyright (c) 2021 Andrew Yang. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Andrew Yang
+-/
+import tactic.elementwise
+import category_theory.limits.shapes.multiequalizer
+import category_theory.limits.constructions.epi_mono
+
+/-!
+# Gluing data
+
+We define `glue_data` as a family of data needed to glue topological spaces, schemes, etc. We
```

Comment:
I have the vague feeling that this should instead be rephrased using coverings in a Grothendieck topology? Does that make sense? I haven't thought hard about this.
Instead of gluing objects in a `glue_data`, isn't the more general theorem something like: if a sheaf for a suitable topology is locally representable, then it is globally representable?

Reply:
The thing is that we do not have the glued space to take cover of in most of my use cases.
The statement you mentioned seems to me to be closer to the comparison lemma (we can glue sheaves on a cover that is somehow dense enough).